lib/carrierwave_data_uri/mount.rb

```diff
@@ -8,10 +8,15 @@ def mount_uploader(column, uploader=nil, options={}, &block)
         super
 
         class_eval <<-RUBY, __FILE__, __LINE__+1
+          attr_accessor :#{column}_data_uri
           attr_accessor :#{column}_data_filename
 
           def #{column}_data_uri=(data)
-            self.#{column} = Parser.new(data).to_file original_filename: self.#{column}_data_filename
+            original_filename = self.#{column}_data_filename.presence
+
+            if data.present?
```

Comment:
This is same to https://github.com/timsly/carrierwave-data-uri/pull/6/files#r54526876.

If use file upload with form post.
There are empty `image_data_uri` attributes.
then, to skip upload process.